apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerExceptionTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright 2021 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Grpc.Core;
+using Xunit;
+
+namespace Google.Cloud.Spanner.Data.Tests
+{
+    public class SpannerExceptionTests
+    {
+        [InlineData(ErrorCode.InvalidArgument, StatusCode.InvalidArgument, "Invalid argument", false)]
+        [InlineData(ErrorCode.Aborted, StatusCode.Aborted, "Transaction aborted", true)]
+        [InlineData(ErrorCode.Unknown, StatusCode.Unknown, "An unknown error occurred", false)]
+        [InlineData(ErrorCode.Unknown, StatusCode.Unknown, "Transaction outcome unknown", true)]
+        [Theory]
+        public void IsRetryableWithErrorCodeAndInnerException(ErrorCode errorCode, StatusCode statusCode, string message, bool isRetryable)
```

Comment:
Not a blocker, but I would prefer if there were three distinct tests for these, one of each way of building the exception, each receiving the appropiate data.

Reply:
I've changed it into three separate test cases.